frontend/src/metabase/collections/containers/CollectionContent.jsx

```diff
@@ -122,12 +146,9 @@ function CollectionContent({
     setIsBookmarked(shouldBeBookmarked);
   }, [bookmarks, collectionId]);
 
-  const onDrop = useCallback(
-    acceptedFiles => {
-      uploadFile({ file: acceptedFiles[0], collectionId });
-    },
-    [collectionId, uploadFile],
-  );
+  const onDrop = acceptedFiles => {
+    saveFile(acceptedFiles[0]);
```

Comment:
is worth having a check if `acceptedFiles[0]` is not defined?

Reply:
I'll double check, but I don't think the library will fire this handler unless there's a valid file